src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -35,10 +35,10 @@ angular
  * There is an example below of how this should look.
  *
  * ### Notes
- * The `md-autocomplete` uses the the [VirtualRepeat](/api/directive/mdVirtualRepeatContainer)
+ * The `md-autocomplete` uses the the [VirtualRepeat](api/directive/mdVirtualRepeat)
```

Comment:
The `virtual-repeat` directive itself does actually not contain any useful information about the way it works.

I intentionally linked to the `VirtualRepeatContainer`, because it is responsible for handling all the repeated blocks and includes all insight information.

The most important reason, is just to link to the Virtual Repeat container because of the common issues like https://github.com/angular/material/issues/9126.

I know it's linked below as well - but this is just to make the developer explicitly aware of the Virtual Repeat Container when submitting new issues regarding the dropdown.

> We can keep it as it is, just wanted to clarify why I linked it to the container instead.